README.md

```diff
@@ -302,13 +310,14 @@ There are three integration points for custom components:
 
 ## Buttons
 
-`{{validated-form}}` also yields a submit button component that can be accessed with `{{f.submit}}`. It takes the following properties:
+`{{validated-form}}` also yields a submit button component that can be accessed with `{{f.submit}}`. You can also use it as a block style component `{{#f.submit}}Test{{/f.submit}}` if you don't want to pass the label as a property. It takes the following properties:
 
-| Name      | Type      | Description                                                                                 |
-| ----      | ----      | -----------                                                                                 |
-| label     | `String`  | The label of the form button.                                                               |
-| type      | `String`  | Type of the button. Default: `button`.                                                      |
-| disabled  | `Boolean` | Specifies if the button is disabled. Default: Automatic integration of `ember-concurrency`. |
+| Name      | Type      | Description                                                                               |
+| ----      | ----      | -----------                                                                               |
+| label     | `String`  | The label of the form button.                                                             |
+| type      | `String`  | Type of the button. Default: `button`.                                                    |
+| disabled  | `Boolean` | Specifies if the button is disabled.                                                      |
+| loading   | `Boolean` | Specifies if the button is loading. Default: Automatic integration of `ember-concurrency` |
```

Comment:
Hmm, is that actually true? In the code it looks more like `loading` is just specifying the CSS class name used when the button is loading.

Reply:
Oh nevermind, i misunderstood the code ^_^

Reply:
Yes it is true. `loadingClass` computed the CSS class name if `loading` is set to true.